lua/autorun/client/TrueCulling.lua

```diff
@@ -8,6 +8,9 @@ function main()
 		for a, ents_ in pairs(allEnts) do
 			if engine.TickCount()%10 == 0 then
 				ents_:SetNoDraw(true)
+				if ents_:GetClass() == "10C_BaseFlex" then
+					continue
```

Comment:
should work if this does what it does in every single other programming language

Reply:
@shiirudev Hang tight, will get to you in a moment. Just going to review your code :+1: